### PR TITLE
updated video custom mapping description

### PR DESCRIPTION
**Pull request by asad-rafter**

### CODE REVIEW

File: metadata/cloudinary-metadata.zip
Comment (yuval-cloudinary) [inline metadata/cloudinary-metadata.zip:1]: Please replace the tags description with the following:
|
#locale#: The current locale of the site, e.g., en_US.
#sku#: The product ID.
#name#: The product name (without spaces).
#color#: The product's color attribute.
#size#: The size attribute of the product.

Comment (yuval-cloudinary) [inline metadata/cloudinary-metadata.zip:1]: Please bump-up the version, to avoid merging conflicts of ZIP files

Comment (asad-rafter) [reply metadata/cloudinary-metadata.zip:1]: what should be the version

**Comment by yuval-cloudinary** [comment]
24.2.4

File: metadata/cloudinary-metadata.zip
Comment (yuval-cloudinary) [reply metadata/cloudinary-metadata.zip:1]: 24.2.4

**Review by yuval-cloudinary — APPROVED** [review]

**Pull request merged**